.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,6 +3,13 @@
 <!-- Please read CONTRIBUTING.md before submitting your pull request -->
 
 * [ ] Pull request is based on the develop branch
-* [ ] Pull request includes a [changelog file](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#changelog)
+* [ ] Pull request includes a [changelog file](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#changelog). The entry should:
+  - Make sense to users. "Fixed a bug that prevented receiving
```

Comment:
too much wrapping. Annoyingly, it needs to all be on one line for github to format it correctly.